server/api/submissions.test.js

```diff
@@ -37,20 +40,20 @@ describe('testing submission endpoints', () => {
     done();
   });
 
-  // it('gets the submission details', async (done) => {
-  //   const res = await request.get(
-  //     `/api/submissions/${testTeamID}/${testSubmissionId}`
-  //   );
-  //   expect(res.statusCode).toEqual(200);
-  //   expect(res.body.attributes.title).toEqual('A Test Hack');
-  //   done();
-  // });
+  it('updates the submission details', async (done) => {
+    const res = await request
+      .patch(`/api/submissions/${testTeamID}/${testSubmissionId}`)
+      .send(newTestSubmission);
+    expect(res.statusCode).toEqual(200);
+    expect(res.body.attributes.title).toEqual('A Test Hack 2');
+    done();
+  });
```

Comment:
This is good, but can you also search the test database to see if the document was properly updated?

Take a look at how Jason implemented this for the Categories test [here](https://github.com/HackRU/sledge/blob/b99f47d2f5931a2418b5954a9f2140e3213c977f/server/api/admin/categories.test.js#L64)